src/renderer/components/Home.js

```diff
@@ -227,6 +234,31 @@ export default function Home ({ onSelectLanguage }) {
   const [update, setUpdate] = useUpdater()
   const { formatMessage: t } = useIntl()
 
+  const [settingsReset, setSettingsReset] = React.useState(false)
+  const [practiceModeOn, setPracticeModeOn] = React.useState(false)
+  const [invite, setInvite] = React.useState(null)
+
+  const resetInvite = useCallback(() => {
+    setInvite(null)
+  }, [setInvite])
```

Comment:
probably somewhat premature to useCallback here?

Reply:
I suppose it is premature, but we should eventually use `usecallback` here when we deal with the invite. I don't think adding it now adds to much overhead, and I think it will be a good reminder for the future.

Reply:
Actually im going to name this handleInvite...to be a little more explicit